pkg/R/RDD.R

```diff
@@ -21,9 +24,16 @@ setClass("PipelinedRDD",
                       prev_jrdd = "jobj"),
          contains = "RDD")
 
-
-setMethod("initialize", "RDD", function(.Object, jrdd, serialized,
-                                        isCached, isCheckpointed) {
+setMethod("initialize", "RDD", function(.Object, jrdd, serializedMode,
+                                        isCached, isCheckpointed, colNames) {
+  # Check that RDD constructor is using the correct version of serializedMode
+  stopifnot(class(serializedMode) == "character")
+  stopifnot(serializedMode %in% c("byte", "string", "row"))
```

Comment:
Yeah `byte` seems reasonable for that reason. FWIW we should try to isolate the uses of these strings by writing a enum-like class or something like that in a future PR.